aimmdb/ingest/eli.py

```diff
@@ -0,0 +1,49 @@
+import numpy as np
+import pandas as pd
+from pathlib import Path
+
+
+def read_metadata_and_header(path: Path):
+    metadata = {}
+    header = ""
+    dat_file = path.open()
+    for line in dat_file:
```

Comment:
There's a faster and more pythonic way to do this without using a for loop. I would recommend using list comprehension instead.